lib/utils.js

```diff
@@ -298,31 +298,25 @@ export function getLangString(langData = "") {
 
 // DAM METADATA
 
-export const useEnhancedImageAsset = (image) => {
-  const globalMetadata = useGlobalData("metadata");
-  const { id } = image || {};
-
-  const { damMetadata } =
-    globalMetadata.find((element) => {
-      const { assetId } = element;
-
-      if (id && assetId) {
-        return id.toString() === assetId.toString();
-      }
-
-      return false;
-    }) || {};
-
-  if (damMetadata) {
-    const { metadataValue: altText } = damMetadata.find(
-      (element) => element.metadataKey === "altText"
-    );
-    const { metadataValue: thumb } = damMetadata.find(
-      (element) => element.metadataKey === "thumbnailUrl"
-    );
-
-    image = { ...image, altText, thumb };
+export const useDamAssetAsImage = (damAsset) => {
```

Comment:
since dam assets could be simple images, gallery images with additional fields, or a non-image I started with a simple hook that parses the dam asset into our image shape